### DISCUSSION TITLE
A UI for building issue forms

### DISCUSSION THREAD

**Discussion by labrose** [community/community#4265] [Projects and Issues]
Configuring issue forms manually can be a chore and prone to errors. I'd like to be able to create issue forms using some kind of graphical interface.

**Comment by geromegrignon** [comment]
Hello @labrose,
While waiting for an official solution, I built an interface to build Issue Forms with a form: https://www.issue-forms-creator.app/new

![image](https://user-images.githubusercontent.com/32737308/123811373-79613b80-d8f3-11eb-9ff4-29ce067016ee.png)

**Reply by lukehefson** [reply]
Whoa this is so cool @geromegrignon!!! 👏

**Reply by AlCalzone** [reply]
That's amazing
